kaldb/src/main/java/com/slack/kaldb/logstore/schema/SchemaAwareLogDocumentBuilderImpl.java

```diff
@@ -357,6 +357,7 @@ public Document fromMessage(LogMessage message) throws JsonProcessingException {
     for (String key : message.source.keySet()) {
       addField(doc, key, message.source.get(key), "", 0);
     }
+    LOG.trace("document {}", doc);
```

Comment:
Updated the message. Left it in since so we can dynamically enable it if needed to debug production issues that would be very hard to debug otherwise.